### DISCUSSION TITLE
so-features-enable required after 2.3.10 upgrade

### DISCUSSION THREAD

**Discussion by greatapoc** [Security-Onion-Solutions/securityonion#2012] [General] [answered]
Standalone install

After upgrading to 2.3.10 everything ran fine until after a reboot where elasticsearch, elastalert, filebeat, logstash and kibana failed to start. The following error was found after trying to start elasticsearch.

[root@securityonion log]# so-elasticsearch-start
=========================================================================
Starting elasticsearch...

This could take a while if another Salt job is running.
Run this command with --force to stop all Salt jobs before proceeding.
=========================================================================
[ERROR   ] Failed to pull securityonion:5000/securityonion/so-elasticsearch:2.3.10-features: Error 404: manifest for securityonion:5000/securityonion/so-elasticsearch:2.3.10-features not found: manifest unknown: manifest unknown

I ran so-features-enable and the new packages were pulled down and everything started fine.

Perhaps some sort of check needs to be added to soup to get it to pull the Features images if enabled? Also on that note how do I disable Features?

**Comment by TOoSmOotH** [comment, marked as answer]
#2018 has been opened to fix soup. 

sed -i 's/features: True/features: False/' /opt/so/saltstack/local/pillar/global.sls on the manager. Then highstate all the things.
